application/event_processor/event_processor.py

```diff
@@ -1,75 +1,198 @@
-from json import dumps
+from json import dumps, encoder
 from logging import getLogger
 from os import environ, getenv
+import json
+import sys
+sys.path.append("../")
 
 from aws_lambda_powertools import Tracer
-from boto3 import client
-from change_request import ChangeRequest
+import boto3
+
 from common.logger import setup_logger
 from common.utilities import is_mock_mode, invoke_lambda_function
-from dos import VALID_SERVICE_TYPES, VALID_STATUS_ID, get_matching_dos_services
+from opening_times import spec_open_times_cr_format, spec_open_times_equal
+from change_request import *
```

Comment:
Can everything be imported from files explicitly

Reply:
Why?

Reply:
This can be imported by star as our 'Black' code linter doesn't accept it. This is important as if they are left in it would stop any developer on MacOS from commit code